oauth.go

```diff
@@ -208,6 +210,15 @@ func (svc *AlbyOAuthService) IndexHandler(c echo.Context) error {
 	sess, _ := session.Get("alby_nostr_wallet_connect", c)
 	sess.Values["app_name"] = appName
 	sess.Save(c.Request(), c.Response())
+	userID := sess.Values["user_id"]
+	if userID != nil {
+		if appName != "" {
+			//auto-create app
+			return c.Redirect(302, fmt.Sprintf("/apps/new?c=%s", appName))
```

Comment:
you need to clean the session here, otherwise the same happens on the next visit

Reply:
No, because we don't check the session for the app name but the query parameter. And if it is set, then the client should always be redirect to the `/apps/new` page imo.